source/Preferences.cpp

```diff
@@ -133,6 +133,8 @@ void Preferences::Load()
 			alertIndicatorIndex = max<int>(0, min<int>(node.Value(1), ALERT_INDICATOR_SETTING.size() - 1));
 		else if(node.Token(0) == "previous saves" && node.Size() >= 2)
 			previousSaveCount = max<int>(3, node.Value(1));
+		else if(node.Token(0) == "alt-mouse turning")
```

Comment:
Just don't check for this so it isn't stuck on for anyone.

Reply:
I agree that the tutorial popup is enough to let players who may have accidentally toggled this on know what's up. Plus, if someone accidentally toggled this on, they'd probably try and figure out how to toggle it off immediately, so most people who have this setting on when updating are likely those who intended to.